packages/neuron-wallet/src/services/syncBlocks.ts

```diff
@@ -0,0 +1,127 @@
+import { getConnection } from 'typeorm'
+import { Script, OutPoint, Cell } from './cells'
+import TransactionsService, { Input, Transaction } from './transactions'
+import InputEntity from '../entities/Input'
+import ckbCore from '../core'
+
+/* eslint no-await-in-loop: "off" */
+/* eslint no-restricted-syntax: "warn" */
+export default class SyncBlocksService {
+  private lockHashesVar: string[]
```

Comment:
The `Var` means `variable`?

Reply:
yes, can't same as `lockHashes`, can't with `_` prefix

Reply:
If we really can't name it `lockHashes` to avoid name conflict, IMO `lockHashList` is probably better than `lockHashesVar`.

Reply:
updated to `lockHashList`